catalog/admin/module/catalog-admin-module-sources/src/main/webapp/js/view/Utils.js

```diff
@@ -62,7 +62,7 @@ function ($,_, Marionette) {
          * Set up the popovers based on if the selector has a description.
          */
         setupPopOvers: function($popoverAnchor, id, title, description) {
-            var selector = ".description[data-title='" + id + "']",
+            var selector = ".description",
```

Comment:
I'm not sure why the data-title attribute filter was added. I checked the templates and the description class is only on the anchor tags that contain the question icon.